provider/github/poster.go

```diff
@@ -196,6 +196,12 @@ func (p *Poster) createReviewRequest(
 						aComments.Config.Name, c.File, c.Line)
 					continue
 				}
+				if ErrFileNotFound.Is(err) {
+					log.Debugf(
+						"skipping comment on a file not part of the diff. analyzer: %s, file %s, line %d",
```

Comment:
do you think it will be better?

```go
log.With(log.Fields{
   "analyzer": aComments.Config.Name,
   "file": c.File,
   "line": c.Line,
}).Debugf("skipping comment on a file not part of the diff")

Reply:
Maybe it also should be a warning?

Reply:
I also prefer the structured form. I just copied the previous logs that we had here, but we can change all of them in this PR :+1:

Reply:
If you don't mind let's change them. Structured form is very useful with search in kibana.

Reply:
done in a598131